federated_learning/financial_fraud_detection/README.md

```diff
@@ -48,6 +48,14 @@ In the privacy-enhanced joint machine learning architecture, each financial inst
 
 ## Quick Start
 
+#### Environment requirement
+
+OS version: MacOS Sonoma 14.1.1
+
+Python version: Python 3.8
+
```

Comment:
Please remove this line as it duplicates line 57.

Reply:
done